Automattic/vip-go-mu-plugins
# Update logstash lib to write to a logfile

## Description

Update the logstash library to write to a file, `/tmp/logstash.log`, instead of an API endpoint as the endpoint is not ready yet.

This emulates behaviour from WPCOM `log2logstash` function

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Checkout PR
1. Sandbox a VIP Go site
1. `wp shell`
1. `\Automattic\VIP\Logstash\log2logstash( ['severity'=>'alert', 'feature'=>'testing', 'message'=>'Test alert'] );`
1. `do_action( 'shutdown' );`
1. Ensure `/tmp/logstash.log` file is created
1. Ensure entry in logstash file is correct
```
$ cat /tmp/logstash.log
{"site_id":1,"blog_id":1,"host":"beruberuberu-test.go-vip.co","severity":"alert","feature":"a8c_vip_testing","message":"Test alert","user_id":0,"extra":"[]","timestamp":"2020-03-31 12:44:26","index":"log2logstash","file":"\/chroot\/var\/www\/wp-content\/mu-plugins\/logstash\/class-logger.php","line":"401"}
```
